specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-04-01-preview/WorkspaceManagerAssignments.json

```diff
@@ -0,0 +1,736 @@
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Job"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Delete a workspace manager job.": {
+            "$ref": "./examples/workspaceManagerAssignments/DeleteJob.json"
+          }
+        },
+        "tags": [
+          "workspaceManagerAssignments"
+        ],
+        "description": "Deletes the specified job from the specified workspace manager assignment",
+        "operationId": "WorkspaceManagerAssignmentJobs_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/2.0/types.json#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceManagerAssignmentName"
+          },
+          {
+            "$ref": "#/parameters/JobName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "WorkspaceManagerAssignment": {
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/AzureEntityResource"
+        }
+      ],
+      "description": "The workspace manager assignment",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/WorkspaceManagerAssignmentProperties",
+          "description": "The workspace manager assignment object",
+          "x-ms-client-flatten": true
+        }
+      },
+      "type": "object"
+    },
+    "WorkspaceManagerAssignmentProperties": {
+      "description": "The workspace manager assignment properties",
+      "properties": {
+        "targetResourceName": {
+          "description": "The name of the workspace manager group targeted by the workspace manager assignment",
+          "type": "string"
+        },
```

Comment:
I wonder if it would be better to have a full resource id of the group? To be sure there's no ambiguity, and to provide a 'clear' picture of what the original target of this reference was, in case there is ever any resource move.

Reply:
(instead of just the resource name)